package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "playwright",
```

Comment:
I would expect something more descriptive. Like Dummy shop TAF or something like that